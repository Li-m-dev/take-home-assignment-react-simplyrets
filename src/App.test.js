import { render } from '@testing-library/react';
import App from './App';
import React from 'react';

it('should render', () => {
  // Act
  const { container } = render(<App />);

  // Assert
  expect(container).toMatchSnapshot();
});
