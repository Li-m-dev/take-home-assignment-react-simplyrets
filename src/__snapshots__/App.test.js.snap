// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div>
    <div
      class="header"
    >
      <h1
        class="header_text"
      >
        Property Listings
      </h1>
    </div>
    <div
      class="list_container"
    />
  </div>
</div>
`;
